dipy/workflows/segment.py

```diff
@@ -91,7 +91,7 @@ def get_short_name(cls):
         return 'recobundles'
 
     def run(self, streamline_files, model_bundle_files,
-            greater_than=50, less_than=1000000,
+            greater_than=30, less_than=1000000,
```

Comment:
Can we avoid to change this default value?

Reply:
ok